components/breadcrumbs/breadcrumb.js

```diff
@@ -0,0 +1,90 @@
+import '../icons/icon.js';
+import '../link/link.js';
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+import { ifDefined } from 'lit-html/directives/if-defined.js';
+import { RtlMixin } from '../../mixins/rtl-mixin.js';
+
+class Breadcrumb extends RtlMixin(LitElement) {
+
+	static get properties() {
+		return {
+			_compact: {
+				attribute: 'compact',
+				type: Boolean,
+				reflect: true
```

Comment:
You might be doing this with the `compact` pass-through changes you're making but just to make sure it doesn't get missed. This doesn't need to be reflected anymore now that its added to the `d2l-breadcrumb-wrapper`.

Reply:
Yeah makes sense

Reply:
Done ✔️